circuits/ts/__tests__/TallyVotes.test.ts

```diff
@@ -74,32 +73,39 @@ describe("TallyVotes circuit", function test() {
       const commands: PCommand[] = [];
       // Sign up and publish
       const userKeypair = new Keypair();
-      stateIndex = BigInt(
-        maciState.signUp(userKeypair.pubKey, voiceCreditBalance, BigInt(Math.floor(Date.now() / 1000))),
-      );
+      maciState.signUp(userKeypair.pubKey, voiceCreditBalance, BigInt(Math.floor(Date.now() / 1000)));
 
       pollId = maciState.deployPoll(
         BigInt(Math.floor(Date.now() / 1000) + duration),
-        maxValues,
+        maxValues.maxVoteOptions,
         treeDepths,
         messageBatchSize,
         coordinatorKeypair,
       );
 
       poll = maciState.polls.get(pollId)!;
-      poll.updatePoll(stateIndex);
+      poll.updatePoll(BigInt(maciState.stateLeaves.length));
+
+      // Join the poll
+      const { privKey } = userKeypair;
+      const { privKey: pollPrivKey, pubKey: pollPubKey } = new Keypair();
+
+      const nullifier = poseidon([BigInt(privKey.rawPrivKey.toString())]);
+      const timestamp = BigInt(Math.floor(Date.now() / 1000));
```

Comment:
Timestamp is used in Poll.ts, what to use instead? What do you recommend?
```
    const stateLeaf = new StateLeaf(pubKey, newVoiceCreditBalance, timestamp);
```

Reply:
if we removed the timestamp check from the circuit (as it's not really needed in the anon poll joining flow i.e. you can only join within the voting deadline so if you are in the tree it means you signed up on time) then we should not need the timestamp anywhere else (so it can be removed from the StateLeaf object), unless there's something I'm missing?

Reply:
Yes, we can remove it but I am just concerned about additional time for removing and testing everything again because timestamp and State Leaf are widely used in the project. Maybe to leave it for M3?

Reply:
ok please make note of it and cleanup in M3